website/docs/tutorial/1c-setting-up-databricks.md

```diff
@@ -5,10 +5,244 @@ id: setting-up-databricks
+
+- Stripe Payments = `stripe_payments` 
+
+- Jaffle Shop Orders = `jaffle_shop_orders`
+
+    <Lightbox src="/img/databricks_tutorial/images/raw_tables_created.png" title="Raw Data Tables" />
+
 ## Enable a SQL endpoint for Databricks
 
+For dbt to connect to the most optimized SQL experience in Databricks, we need to create an SQL Endpoint for our connection.
+
+1. Up to this point, we have been working in the Data Science & Engineering space. Now we need to change over to SQL.  Find the drop down menu and toggle into the SQL space.
+
+<Lightbox src="/img/databricks_tutorial/images/go_to_sql.png" title="SQL space" />
+
+2. We will be setting up a SQL endpoint now.  Select `SQL Endpoints` from the left hand side console.  You will see that a default SQL Endpoint exists.  
+
+<Lightbox src="/img/databricks_tutorial/images/sql_endpoints.png" title="SQL Endpoints" />
+
+3. Click `Start` on the Starter Endpoint.  This will take a few minutes to get the necessary resources spun up.
+
+4. Once that's done, let’s ensure that we can query the training data that we loaded earlier.  Navigate back to the SQL space through the left hand menu and choose `SQL Editor`.  This will bring you to a query editor.
+
+5. Ensure that you can run a select * from each of the tables with the following code snippets. 
+
+```sql
+select * from default.jaffle_shop_customers
+```
+
+```sql
+select * from default.jaffle_shop_orders
+```
+
+```sql
+select * from stripe.payments
+```
+
+<Lightbox src="/img/databricks_tutorial/images/query_check.png" title="Query Check" />
+
+6. To get setup in dbt Cloud, you will now need to get the SQL Endpoint connection information and generate a user token. 
+
+You can find your SQL endpoint connection information by going to the `Databricks UI > SQL > SQL Endpoints > Starter Endpoint > Connection details`. Save this information or keep this window open for the next section.
+
+<Lightbox src="/img/databricks_tutorial/images/SQL_Endpoint_Details.png" title="Databrick SQL Endpoint Connection Information" />
+
+7. To generate a user token for your development credentials in dbt Cloud, click on `Settings` on the left side console (while still in the SQL part of the workspace). Click on `Personal Access Token` and provide a comment like `dbt Cloud development`. Save the token information somewhere because you will need it for the next part. 
+
+<Lightbox src="/img/databricks_tutorial/images/Generate_user_token.png" title="Generate User Token" />
+
+
+Congratulations! At this point, you have created a Databricks account, loaded training data, and successfully set up a SQL end point to query the data.  
+
 ## Set up a project to connect dbt Cloud to Databricks
```

Comment:
note for breaking changes: I made the same comment on Pat's PR.  This content is solid, but we should make sure the rest of the GS tutorial is smooth and we don't repeat ourselves.

Reply:
GS tutorial?

Reply:
GS Tutorial = Getting Started Tutorial.  Since this is nested in the GS tutorial, we likely want to update the Set up project part of the overall tutorial. 
<img width="309" alt="image" src="https://user-images.githubusercontent.com/10554338/152820588-641954ea-e855-4e47-9112-04ba821abd68.png">

Reply:
I think it makes more sense as a landing page/overview rather than with actual instructions

Reply:
correction to my original response Create* a project.

The final section of the three WH set up guides covers the same thing as Create a project / dbt Cloud. So I'm just making a note that we need to tweak things so this moves smoothly into Build your first models.

Reply:
nope! good to go and we will handle on the final merge to prod.